### DISCUSSION TITLE
How to ensure markdown content is responsive tex equations

### DISCUSSION THREAD

**Discussion by spiritoftime** [fuma-nama/fumadocs#852] [Q&A] [answered]
Hello! I realised that tex equations may end up overflowing on smaller sized screens. Now I could add media queries etc to force this one equation to be responsive, but I was curious if there are other inbuilt ways fumadocs alr has (will probably try throwing it in a callout or something later). Thank you!

![Screenshot_2024-09-12-16-27-55-456_com android chrome](https://github.com/user-attachments/assets/1d98f541-024c-47ae-bade-631d26b556a8)

Markdown content:

You can be set to earn 300million dollars in 30years, but you might owe ah long 1million dollars and you need to pay within a week. Liquidity is how much money you can source in a short period of time (you can sell all your assets like stocks, etc). ```math \text{Net Working Capital} = \text{Current Assets} - \text{Current Liabilities} ``` Just bear with me - this is an introductory lecture and there will be a example walkthrough at the bottom.

https://github.com/spiritoftime/psle-math/blob/main/open-math-prep/content/finance/incomeBal.mdx

**Comment by fuma-nama** [comment]
wrap the codeblock in a div with `overflow: auto` (and style it as you want)

**Reply by spiritoftime** [reply]
okay nvm i'm dumb to even think of media queries (been too spoiled by shadcn etc), but could you take some time to enlighten me why the tex even overflows, when its already in the layout container

thanks!

**Reply by fuma-nama** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/25b5d019-ecf5-448d-a7d8-8ff5dc5b9204)

there's a `katex-display` class, put your styling (e.g. `overflow: auto`) to it

**Reply by spiritoftime** [reply]
thanks for constantly entertaining my dumbass qns!=)
